serving/docker/deepspeed.Dockerfile

```diff
@@ -15,10 +15,10 @@ ARG djl_version=0.23.0~SNAPSHOT
 ARG python_version=3.9
 ARG torch_version=2.0.1
 ARG torch_vision_version=0.15.2
-ARG deepspeed_wheel="https://publish.djl.ai/deepspeed/deepspeed-0.9.2-py2.py3-none-any.whl"
+ARG deepspeed_wheel="https://publish.djl.ai/deepspeed/deepspeed-nightly-py2.py3-none-any.whl"
 ARG lmi_dist_wheel="https://publish.djl.ai/lmi_dist/lmi_dist-nightly-py3-none-any.whl"
 ARG protobuf_version=3.20.3
-ARG transformers_version=4.29.2
+ARG transformers_version=4.30.1
 ARG accelerate_version=0.19.0
 ARG diffusers_version=0.15.0
```

Comment:
Can you check if we can support later versions?

Reply:
I will do a local build and test if we can move everything to latest. I will move this into draft until I have finished.